ocaml/src/asd_config_test.ml

```diff
@@ -0,0 +1,55 @@
+(*
+Copyright (C) 2016 iNuron NV
+
+This file is part of Open vStorage Open Source Edition (OSE), as available from
+
+
+    http://www.openvstorage.org and
+    http://www.openvstorage.com.
+
+This file is free software; you can redistribute it and/or modify it
+under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+as published by the Free Software Foundation, in version 3 as it comes
+in the <LICENSE.txt> file of the Open vStorage OSE distribution.
+
+Open vStorage is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY of any kind.
+*)
+
+open OUnit
+open Asd_config
+let test_basic_config () =
```

Comment:
Maybe I misunderstood, but I thought this test would put cfg/asd_demo.json under test?
Fwiw there's still an error in asd_demo.json (the 1_000_000 we talked about earlier, which was fixed on your branch, but then undone again when merging latest master?)